docs/user-docs/latest/plugins/installing-plugins-via-marketplace.md

```diff
@@ -4,37 +4,41 @@ description: Instructions to install new plugins in a Strapi application via the
-In the meantime, plugins can be installed via the Command Line Interface (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
-:::
+The Marketplace is where users can find additional plugins to customize Strapi applications. The Marketplace is located in the admin panel, indicated by ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_. In the Marketplace, users can browse or search for plugins, link to detailed plugin descriptions and submit new plugins. This documentation describes how to navigate the Marketplace and how to install Marketplace plugins.
 
-The Marketplace is a section of the admin panel that lists all plugins that can be installed in a Strapi application. The Marketplace is accessible from ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_ in the main navigation of the admin panel.
+ ::: strapi
+The Marketplace in the admin panel only displays v4 plugins, but all plugins for all Strapi versions are discoverable in the [Strapi Market](market.strapi.io). Keep in mind that v3 and v4 plugins are not cross-compatible.
+:::
 
-<!--
+![The Marketplace interface](../assets/plugins/marketplace-v4.png)
 
-::: note
-Plugins can also be installed via the Command Line Interface (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
-:::
+Plugins are displayed on individual cards containing:
 
-![Interface of the Marketplace](../assets/plugins/marketplace.png)
+- the plugin name, sometimes followed by either of the following badges:
+   - ![made by Strapi icon](../assets/icons/official-market.svg) to indicate the plugin is made by Strapi,
+   - ![verified by Strapi icon](../assets/icons/verified-marketplace.svg) to indicate the plugin was verified by Strapi.
+- the plugin description
+- a **Learn more** button for additional information, including detailed implementation instructions
+- a **Copy install command** button to copy the plugin installation command to the local clipboard
 
-The Marketplace displays each available plugin in a box, which contains:
-- the name of the plugin,
-- the description of the plugin,
-- indications on the current status of the plugin:
-   - "Compatible with your app": indicates that the plugin is not installed yet but can be installed on your Strapi application
-   - "Already installed": indicates that the plugin is already installed and available in your Strapi application
+In the top right corner of the Marketplace, the **Submit your plugin** button redirects to the Strapi Market where it is possible to submit your own plugin.
 
 ::: tip
-Click on the link icon ![External link icon](../assets/icons/external_link.svg) next to the name of a plugin to be redirected to the plugin package in the Strapi GitHub repository.
+The search bar displays find as you type results that search the plugin name and description.
```

Comment:
Not sure about this sentence, what's the meaning you meant to convey? Would adding dashes ("find-as-you-type") be enough?